src/Auth/wwwroot/css/site.css

```diff
@@ -0,0 +1,82 @@
+﻿body {
```

Comment:
Whole file needs review and updating to the CSS standards. Specifically, newlines between rules are missing. If this is a compiled file generated from the .less, it should be minified.

Reply:
Should eventually be ported to SCSS but we can pull this as vanilla CSS and do that later, along with integrating the linter. Please just make sure it conforms to the documented standards :)